olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -307,6 +308,17 @@ NetworkCurl::NetworkCurl(NetworkInitializationSettings settings)
                         static_cast<int>(error));
   }
 
+  if (curl_log_path_) {
+    stderr_ = fopen(curl_log_path_->c_str(), "w+");
+    if ( stderr_ == nullptr ) {
+      OLP_SDK_LOG_ERROR(
```

Comment:
Maybe warning as it is not crucial for the functionality

Reply:
Will fix in the next commit